src/late_multicellular_stage/editor/MetaballEditorComponentBase.cs

```diff
@@ -565,7 +565,7 @@ protected void OnSymmetryPressed()
         */
     }
 
-    /// <inheritdoc cref="HexEditorComponentBase{TEditor,TCombinedAction,TAction,THexMove}.OnHexEditorMouseEntered"/>
+    /// <inheritdoc cref="HexEditorComponentBase{TEditor,TCombined,TAction,THexMove,TContext}.OnHexEditorMouseEntered"/>
```

Comment:
I think I fixed this on my branch where I fix various other things. I think I fixed this at least a bit better as I kept the right `TCombinedAction` name here. So maybe you can just revert this change and leave it be for a bit (I'm hopefult it won't take more than two weeks to get my changes regarding this merged)?